src/CONST.js

```diff
@@ -64,6 +64,7 @@ const CONST = {
         HOMEPAGE_REPORTS_LOADED: 'homepage_reports_loaded',
         SWITCH_REPORT: 'switch_report',
         COLD: 'cold',
+        REPORT_ACTION_ITEM_LAYOUT_DEBOUNCE_TIME: 1000,
```

Comment:
NAB, but is it true that a slow child component taking longer than the denounce time to render would trigger a false measurement?

Reply:
Yes, I think if we had a child that takes more than a second to render we'd definitely get a funky measurement.

I think it's pretty unlikely for that to happen.

I came across another issue here though that I'm going to look into addressing now.

If we switch chats before thing are done rendering it's possible to get a negative value recorded so I'll look into how to prevent this before proceeding.

Reply:
Ok figured that one out and we just need to cancel the debounced method in `componentWillUnmount()` so it does not incorrectly set the timer end when switching to the next chat.

So, do you think it's OK to have this... or should we perhaps increase the time?

>a slow child component taking longer than the denounce time to render would trigger a false measurement

Reply:
As unlikely as this is, I think it would be better to set a slightly higher debounce time -- _just in case_. But it's NAB. I'll leave it up to you 👍